ruby-server-example/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Ruby Alpine image (after using bsf dockerfile digest)
+FROM ruby:3.1-alpine AS build
+
+# Install all required dependencies for Ruby gems
+RUN apk add --no-cache build-base libxml2-dev libxslt-dev tzdata
```

Comment:
We shouldn't need this as all OS dependencies should come from base image built by bsf.